src/utils/src/types/work-package-types.ts

```diff
@@ -49,4 +49,52 @@ export const workPackageInputSchemaBody = {
   ]
 } as const;
 
-export type CreateWorkPackagePayload = FromSchema<typeof workPackageInputSchemaBody>;
+export type CreateWorkPackagePayload = FromSchema<typeof workPackageCreateInputSchemaBody>;
+
+export const workPackageEditInputSchemaBody = {
+  type: 'object',
+  properties: {
+    wbsElementId: { type: 'integer', minimum: 0 },
+    userId: { type: 'integer', minimum: 0 },
+    name: { type: 'string' },
+    crId: { type: 'integer', minimum: 0 },
+    projectId: { type: 'number', minimum: 0 },
+    startDate: { type: 'string', format: 'date' },
+    duration: { type: 'number', minimum: 0 },
```

Comment:
Make sure to use integer if you don't want to allow decimals, Check the database schema to make sure you're specifying the right type.

Reply:
Did this get resolved? Duration is intended to be an integer

Reply:
oh for some reason I thought a duration could be a decimal woops